src/actions/paymentAction.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import axios from "axios";
+
+class PaymentAction extends React.Component {
```

Comment:
This does not need to be a class component. It can be an object with functions that you export.